### ISSUE TITLE
Certify data usage on the Privacy practices for Chrome Extension

### ISSUE THREAD

**Issue by steventsao** [emberjs/ember-inspector#1510]
**Describe the bug**
Another error blocking Build & Publish in Actions. This is important to fix to unblock the next release. Please have those with the appropriate permission certify data usage to fix the following error:

cc @rwwagner90  @rwjblue @chancancode 
```
"{
  "error": {
    "code": 400,
    "message": "Publish condition not met: To publish your item, you must certify that your data usage complies with our Developer Program Policies. You can certify this on the Privacy practices tab of the item edit page of the Developer Dashboard.",
    "errors": [
      {
        "message": "Publish condition not met: To publish your item, you must certify that your data usage complies with our Developer Program Policies. You can certify this on the Privacy practices tab of the item edit page of the Developer Dashboard.",
        "domain": "chromewebstore.access",
        "reason": "badRequest"
      }
    ]
  }
}
"
```
**To Reproduce**
https://github.com/emberjs/ember-inspector/runs/2044019465?check_suite_focus=true

**Expected behavior**
No error on Build & Publish

**Screenshots**
<img width="1035" alt="Screen Shot 2021-03-05 at 6 03 39 PM" src="https://user-images.githubusercontent.com/15185301/110191413-22cc7080-7ddd-11eb-80ca-053ab0728be0.png">

**Environment**
Ember version, browser, etc.

**Additional context**
https://github.com/emberjs/ember-inspector/pull/1509

**Comment by steventsao** [comment]
Pinging more people who might have access from the committee. 
![Screen Shot 2021-03-08 at 11 01 27 AM](https://user-images.githubusercontent.com/15185301/110368574-15420100-7ffe-11eb-8ea0-41fd17c420e1.png)


@tomdale @MelSumner

**Comment by rwwagner90** [comment]
<img width="747" alt="Screen Shot 2021-03-09 at 4 02 24 PM" src="https://user-images.githubusercontent.com/2640861/110537557-d8a30200-80f0-11eb-9f10-5548587eff66.png">

@steventsao I went in and checked these boxes. Is that all we needed?

**Comment by steventsao** [comment]
Thanks @rwwagner90 ! I don't think you need to check these boxes every time. They look more like a one-time agreement that you have to publish like those in React dev tool:

Actual
![Screen Shot 2021-03-09 at 3 53 13 PM](https://user-images.githubusercontent.com/15185301/110554412-9d540380-80ef-11eb-8d05-ff6f5c863cd7.png)

Expected privacy page
![Screen Shot 2021-03-09 at 3 53 06 PM](https://user-images.githubusercontent.com/15185301/110554415-9e853080-80ef-11eb-804d-82cc293a56e7.png)
https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en

**Comment by steventsao** [comment]
Looks like we got a successful build! @rwwagner90 
I'm looking into why the Chrome store hasn't picked up the release yet. Could it be another manual step?
https://developer.chrome.com/docs/extensions/mv3/hosting/#:~:text=To%20release%20an%20update%20to,version%22%20field%20of%20the%20manifest.&text=Convert%20the%20updated%20extension%20directory,new%20package%2C%20and%20hit%20Publish.

**Comment by chancancode** [comment]
There _is_ an automatic “nightly” version of the inspector at an unlisted URL, but that still requires approval from someone at google before it’s “published” so often there are huge backlogs/delays. As far as the main version that everyone uses, that is only published on pushing a git tag (and then someone from google will eventually approve that and the chrome store listing will be updated). 

TL;DR we will need to push a tag. If that was already done successfully then we are waiting for a reviewer at google to approve. 

Is the inspector something that you would like to help maintain?

**Comment by steventsao** [comment]
I can try! I don't have a deep understanding of the tool yet, but I can triage few open issues right now! @chancancode

**Comment by rwwagner90** [comment]
You've been very helpful so far @steventsao! 😃

**Comment by chancancode** [comment]
@steventsao awesome! after emberconf, we can try to restart the ember-inspector meetings (again) and go over the areas that we could use help in
